client/components/Users.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import {connect} from 'react-redux';
+// import { getUsers } from '../redux/';
+import faker from 'faker';
+
+export class Users extends React.Component {
+    constructor(props){
+        super(props);
+        this.state = {
+            users: [],
+            adminUsers: []
+        }
+        this.handleChecked = this.handleChecked.bind(this)
+        this.handleDeleteUser = this.handleDeleteUser.bind(this)
+        this.handleResetPassword = this.handleResetPassword.bind(this)
+
+    }
+
+    componentDidMount(){
+        // I'll change the code below once we confirm the users data
+        let tempArrOfUsers = []; // Added 10 users from faker as dummy data
+        while(tempArrOfUsers.length < 11){
+            tempArrOfUsers.push(faker.name.firstName())
+        }
+        this.setState({users: tempArrOfUsers})
+    }
+
+
+    async handleDeleteUser(ev) {
+        // Changed the local state. I'll update the store once the users data is confirmed
+        if(this.state.adminUsers.includes(ev.target.name)){
+            alert('Change admin status before deleting')
+        } else {
+            console.log(`${ev.target.name} deleted`)
+            const updatedUsers = await this.state.users.filter( u => u !== ev.target.name)
+            this.setState({users: updatedUsers})     
+        }
+        //In case I need to use it later 
+            /* let rowNumber = ev.target.value;
+            document.getElementById("table").deleteRow(rowNumber + 1)*/
+    }   
+
+    handleChecked(ev){
+        //Needs to be integrated with backend
+        if(ev.target.checked) {
+            const updatedAdminUsers = [...this.state.adminUsers, ev.target.name]
+            this.setState({adminUsers: updatedAdminUsers})
+            console.log(`${ev.target.name} is now admin`);
+        }
+        else {
+            const updatedAdminUsers = this.state.adminUsers.filter( u => u !== ev.target.name)
+            this.setState({adminUsers: updatedAdminUsers});
+            console.log(`${ev.target.name} is no longer an admin`)
+        }
+    }
+
+    handleResetPassword(ev){
+        //Needs to be integrated with backend
+        alert("Needs to be integrated with backend")
+    }
+
+   render(){
+       let id = 0; // will change id to pk when users data is confirmed
+       return (
+           <div>
+               <h1>Users</h1>
+               <div className="adminNav">
```

Comment:
If you take a look at Adam's PR, we'll want to have the Admin navigation options show up in the top black nav bar. Likely "COLLECTION" would be the same as "PRODUCTS", we'd just insert add/edit/delete buttons on the existing AllArtwork component.

Reply:
I'll take a look into his PR